fitting/fitScanData.py

```diff
@@ -68,7 +74,7 @@ def feed(self, event):
                 if(event.vcal > 254):
                     self.scanCount[event.vfatN][event.vfatCH] += event.Nhits
             else:
-                charge = self.calDAC2Q_m[event.vfatN]*(256-event.vcal)+self.calDAC2Q_b[event.vfatN]
+                charge = self.calDAC2Q_m[event.vfatN]*(event.vcal)+self.calDAC2Q_b[event.vfatN]
                 if((256-event.vcal) > 254):
```

Comment:
Hi Brian, I am (very slowly) still looking into bug #61 and so have been converting charge to chargebin later in this script. Here should the 256 not also be removed from this line? So it becomes identical to L74?

Reply:
In this particular case no.  We are trying to count the number of `scurves` that have been processed.  So all we are interested in is the raw register value.  In the case of voltage step mode (e.g. `isCurrentPulse == False`) the height of the cal pulse is high when the register value is low.